tests/integration/targets/aws_api_gateway_domain/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# defaults file for aws_api_gateway_domain
+
+api_gateway_domain_name: "{{ resource_prefix }}-api-gw-test.foobar.com"
+api_gateway_domain_tls_arn: "arn:aws:acm:{{ aws_region }}:{{ aws_account_id }}:certificate/abc123-acb123-abc123-abc123"
```

Comment:
The tests still needs a real ACM TLS cert for a given domain. Otherwise we have to disable it on CI, as creation of the API GW domain setup will of course validate the cert first.